cmd/registry/cmd/resolve/resolve.go

```diff
@@ -64,19 +64,15 @@ func Command(ctx context.Context) *cobra.Command {
 
 			log.Printf("Generated %d actions. Starting Execution...", len(actions))
 
-			taskQueue := make(chan core.Task, 1024)
-			for i := 0; i < 64; i++ {
-				core.WaitGroup().Add(1)
-				go core.Worker(ctx, taskQueue)
-			}
-			defer core.WaitGroup().Wait()
-			defer close(taskQueue)
-
 			// Submit tasks to taskQueue
-			for i, a := range actions {
-				taskQueue <- &controller.ExecCommandTask{
-					Action: a,
-					TaskID: fmt.Sprintf("task%d", i),
+			for _, a := range actions {
+				log.Printf("Executing action: %s", a)
+				rootCmd := cmd.Root()
+				rootCmd.SetArgs(strings.Split(a, " "))
+				log.Printf("%s", rootCmd.CalledAs())
+				err = rootCmd.Execute()
+				if err != nil {
+					log.Printf("Error executing action: %s, %s", a, err.Error())
```

Comment:
No action needed, but FYI errors are automatically unwrapped when we use `%s`. This has the same behavior:
```
if err := rootCmd.Execute(); err != nil {
    log.Printf("Error executing action: %s, %s", a, err)
}
```

We also log the action right before this error check, so the action itself doesn't need to be included. As long as it doesn't make the error hard to understand it's also fine to keep it there.

Reply:
Done.